scarb/src/core/resolver.rs

```diff
@@ -43,6 +43,15 @@ impl Resolve {
             .unique()
             .collect_vec()
     }
+
+    /// Collect `[PackageId`]s of all directed dependencies of the package.
+    pub fn package_dependencies(
+        &self,
+        package_id: PackageId,
+    ) -> impl Iterator<Item = PackageId> + '_ {
+        self.graph
+            .neighbors_directed(package_id, petgraph::Direction::Outgoing)
+    }
 }
```

Comment:
Not used?

Reply:
It is here though 🤔 
https://github.com/software-mansion/scarb/blob/1c1f0766a36896c5d8193c7462f1d4d4bf6789d5/scarb/src/core/lockfile.rs#L86